src/AppInstallerCommonCore/FileLogger.cpp

```diff
@@ -24,6 +25,14 @@ namespace AppInstaller::Logging
         m_name = GetNameForPath(filePath);
         m_filePath = filePath;
 
+        // Prevent inheritance to ensure log file handle is not opened by other processes.
+        FILE* filePtr;
+        if (!fopen_s(&filePtr, m_filePath.string().c_str(), "w") && filePtr != NULL)
+        {
+            SetHandleInformation((HANDLE)_get_osfhandle(_fileno(filePtr)), HANDLE_FLAG_INHERIT, 0);
+            std::ofstream outFile(filePtr);

```

Comment:
There are two functions that open the stream; a helper should be used to update both.

This code is creating `outfile` and then immediately throwing it away.  You would want:
```
m_stream = std::ofstream{ filePtr };
```
And then to not call `m_stream.open`.